### ISSUE TITLE
[Package]: ibus-daemon

### ISSUE THREAD

**Issue by ask9027** [termux/termux-packages#17572]
### Why is it worth to add this package?

IBus  is  an  Intelligent Input Bus. It is a new input framework for Linux OS. It provides full featured and user friendly input method user interface.  It also may help  developers to develop input method easily.

to use multiple input method while in GUI mode

### Source code URL

https://github.com/ibus/ibus

### Packaging policy acknowledgement

- [X] I certify that I have read [Termux Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy) and understand that my request will be denied in case of violation.

**Comment by phucvinh52** [comment]
+1

**Comment by PChaicot** [comment]
oh wait...
package is already available in the x11 repository

**Comment by robertkirkman** [comment]
Yes I added ibus , and then someone else very smart fixed it I think to work better!

please try it I hope it works. some functionality might not work.
